server.js

```diff
@@ -28,10 +48,37 @@ app.get('/api/*', (req, res) => {
     res.set(r.headers)
     res.send(r.data)
   }).catch(e => {
+    // console.error(e)
     res.status(e.response.status).end()
   })
 })
 
+app.get('/*', (req, res) => {
+  // match the routes to the url
+  match({ history, routes, location: req.url }, (err, redirect, props) => {
+    // `RouterContext` is what the `Router` renders. `Router` keeps these
+    // `props` in its state as it listens to `browserHistory`. But on the
+    // server our app is stateless, so we need to use `match` to
+    // get these props before rendering.
+
+    const filters = {
+      ...props.location.query,
+      ...props.params,
+    }
+    const action = updateApp(filters)
```

Comment:
Nope, it was a relic of trying to get the data requisite for the view before returning. But we've agreed that isn't a requirement for merging this, so I have removed it